presto-native-execution/scripts/release-centos-dockerfile/etc/config.properties.template

```diff
@@ -1,3 +0,0 @@
-presto.version=0.279
-discovery.uri=http://127.0.0.1:8080
```

Comment:
Wondering why we don't need something like this anymore.

Reply:
The configs and deployment are part of the orchestration layer.  These will be added on top of the native runtime image.
We have a helm project that could add these templates.
I will ask the release team to publish another image with the Java package (coordinator) added to the native runtime image along with these configs so that users can evaluate Prestissimo.